HerPortal.ManagementShell/CommandHandler.cs

```diff
@@ -159,14 +159,14 @@ public void MigrateAdmins()
                 consortiumCode => consortiumCodeToConsortiumNameDict[consortiumCode]);
 
             var custodianCodesToRemove =
-                adminAction.GetOwnedCustodianCodesInConsortia(user, consortiumCodesUserShouldOwn);
+                adminAction.GetCustodianCodesInConsortia(consortiumCodesUserShouldOwn).ToList();
             outputProvider.Output("To make this user a Consortium Admin, the following LAs will be removed:");
```

Comment:
I suppose we're not removing the LAs, rather the explicit association with the user. Can we reword to be a bit clearer. Don't want to panic someone!